internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/VeniceServerWrapper.java

```diff
@@ -224,8 +223,7 @@ static StatefulServiceProvider<VeniceServerWrapper> generateService(
           .put(MAX_ONLINE_OFFLINE_STATE_TRANSITION_THREAD_NUMBER, 100)
           .put(SERVER_NETTY_GRACEFUL_SHUTDOWN_PERIOD_SECONDS, 0)
           .put(PERSISTENCE_TYPE, ROCKS_DB)
-          .put(ROCKSDB_PLAIN_TABLE_FORMAT_ENABLED, true)
-          .put(ROCKSDB_OPTIONS_USE_DIRECT_READS, false) // Required by PlainTable format
+          .put(SERVER_DATABASE_CHECKSUM_VERIFICATION_ENABLED, true)
```

Comment:
Hmm... we should not leave PT completely untested either, though... Ideally we need to test both, though probably not on every test.

One idea I have in mind is to tweak the cluster wrapper so that whenever we request a cluster with more than 1 server, then the first one will be block-based, as configured here, and the rest will be PT. The cluster wrapper would be the one passing this override, so it would be pretty well contained and there wouldn't be a need to touch a bunch of tests... WDYT?

Reply:
Fixed. Looks like I also need unit test coverage for my one line change.